doc/sphinx-guides/source/user/account.rst

```diff
@@ -155,19 +155,25 @@ The My Data section of your account page displays a listing of all the Dataverse
 
 You can use the Add Data button to create a new Dataverse collection or dataset. By default, the new Dataverse collection or dataset will be created in the root Dataverse collection, but from the create form you can use the Host Dataverse collection dropdown menu to choose a different Dataverse collection, for which you have the proper access privileges. However, you will not be able to change this selection after you create your Dataverse collection or dataset.
 
+.. _account-notifications:
+
 Notifications
 -------------
 
-Notifications appear in the notifications tab on your account page and are also displayed as a number next to your account name.
+Notifications appear in the notifications tab on your account page and are also displayed as a number next to your account name. You also receive notifications via email.
+
+If your admin has enabled the option to change the notification settings, you will find an overview of the notification and email settings in the notifications tab. There, you can select which notifications and/or emails you wish to receive. If certain notification or email options are greyed out, you can’t change the setting for this notification because the admin has set these as never to be muted by the user. You control the in-app and the email notifications separately in the two lists.
```

Comment:
It would be nice if users could view and modify their notification settings via API.

This could, of course, be a future pull request, but if there's interest in adding this now, here's where the notifications API has been stubbed out: https://github.com/IQSS/dataverse/blob/v5.10.1/src/main/java/edu/harvard/iq/dataverse/api/Notifications.java

This issue is related:

- #8487

Having this API would also help with automated testing! 😄

What do you think? Again, it's fine to say that now's not the time. 😄